cdap-app-fabric/src/main/java/co/cask/cdap/app/store/Store.java

```diff
@@ -292,6 +294,21 @@ void changeFlowletSteamConnection(Id.Program flow, String flowletId, String oldV
     throws OperationException;
 
   /**
+   * Adds a schedule for a particular program
```

Comment:
Should say, will throw RuntimeException if the schedule with same name already exists.